en/components/data-chart-legends.md

```diff
@@ -2,7 +2,7 @@
 title: Data Chart | Visualization Tools | Ignite UI for Angular | Infragistics | Legends
 _description: Create a data chart that displays multiple instances of visual elements in the same plot area in order to create composite chart views.
 _keywords: data chart, Ignite UI for Angular, Infragistics
-mentionedTypes: ['XamDataChart', 'Infragistics.Controls.Charts.Legend']
+mentionedTypes: ['XamDataChart', 'XamLegend','Infragistics.Controls.Charts.Legend']
```

Comment:
Should this be `XamDataChart`, `XamLegend`?

Reply:
I believe XamDataChart is correct, but XamLegend likely not. As mentioned in another comment, we use the C# type names - the legend in our C# products is not "XamLegend" - it is just "Legend"

Reply:
yes, that should be Legend